### ISSUE TITLE
does "enable_ip_defrag" works with zbalance (i40e)?

### ISSUE THREAD

**Issue by bth-kumar** [ntop/PF_RING#610]
I used trial version of PF_RING with zc driver (i40e) . I found some packet drops in captured data of tcp sessions but was not coming in pf_ring stats. Captured session with all smaller packets was drop free.

**Comment by cardigliano** [comment]
How did you find this packet loss if stats are not reporting it?
Please note enable_ip_defrag is not available with ZC as it is based on kernel support (standard driver only).

**Comment by bth-kumar** [comment]
Thanks for quick reply, I modified pfcount a little to write session(four-tuple) wise pcap files, which was listening zc:101@0 interface. By looking on the session files with wireshark, I found that there was drops (previous segment not captured) in some files(with only tcp sessions having bigger packet size). It can be the problem with the source.  So I want to cross check it from my side . Could it be the issue with ip fragment?
